docs/config.rst

```diff
@@ -333,3 +339,31 @@ the role using :meth:`sssd_test_framework.utils.sssd.HostSSSD.import_domain`.
 The example above will add the given options to ``sssd.conf``, these are
 required by the client to successfully connect to the IPA server. The keytab
 paths are local paths on the client host.
+
+.. _config-auditd:
+
+Auditd and automatic AVC denials detection
+==========================================
+
+Linux-based roles has access to Auditd utility that automatically collects
+audit.log when a test ends. It can also automatically detect AVC denials and
+fail the test if required conditions are met.
+
+It is possible to set that AVC detection mode to one of ``fail`` (test is mark
```

Comment:
I would like to see also ```error```, see comments in https://github.com/next-actions/pytest-mh/pull/61

Reply:
This was resolved in pytest-mh.